src/ca65/main.c

```diff
@@ -334,6 +334,9 @@ static void SetSys (const char* Sys)
             CBMSystem ("__CX16__");
             break;
 
+        case TGT_SYM1:
+            break;
```

Comment:
Please don't get me wrong, but have you even looked at the case'es above? Why didn't you add

```
NewSymbol ("__SYM1__", 1);
```

Reply:
**FIXED**.  I can't believe I missed that.  Please check and confirm.